### DISCUSSION TITLE
RTSP protocol is Reverse TCP?

### DISCUSSION THREAD

**Discussion by rurusungoa** [bluenviron/gortsplib#570] [Questions]
hello
I would like to inquire about the RTSP communication protocol.
When the communication protocol is Reverse, is the communication method TCP?

I would like to inquire whether you used the TCP protocol when doing the project.

**Comment by aler9** [comment]
Hello, in RTSP the handshake is always performed with TCP, while the streaming is performed with a protocol that is chosen during the handshake (either UDP, UDP-multicast or TCP). You can set this protocol (transport) by setting the `Transport` flag inside `Client`.
